test/integration/stats_integration_test.cc

```diff
@@ -268,6 +268,7 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeClusterSizeWithFakeSymbolTable) {
   // 2019/11/15  9040     43403       44000   upstream: track whether cluster is local
   // 2019/12/10  8779     42919       43500   use var-length coding for name length
   // 2020/01/07  9069     43413       43500   upstream: Implement retry concurrency budgets
+  // 2020/01/08  8889     43477       43500   api: add UpstreamHttpProtocolOptions message
```

Comment:
I'm pretty sure there are merge issues here as I don't think there should be any stats changes for this PR. There weren't previously. Can we wait until master is green and then try reverting these changes?

/wait

Reply:
@mattklein123 adding a new field in Cluster message will change the memory value here, so a change here is expected.

Reply:
Right sorry this wasn't here originally but then we added the field, OK.